dash.el

```diff
@@ -357,7 +357,7 @@ See also: `-each-indexed'."
        (nreverse ,r))))
 
 (defun -map-when (pred rep list)
-  "Return a new list where the elements in LIST that does not match the PRED function
+  "Return a new list where the elements in LIST that do not match the PRED function
```

Comment:
Yes, it can be a little restrictive - especially considering verbosity (for clarity) is often encouraged in naming and descriptions.